src/provider/controller/ExpiryController.ts

```diff
@@ -89,8 +89,9 @@ export class ExpiryController {
             this._nextExpiry = null;
         }
 
-        new ExpireNotification(storedNotificaiton).dispatch(this._store);
-        this.scheduleEarliestExpiry(now);
+        new ExpireNotification(storedNotificaiton).dispatch(this._store).then(() => {
+            this.scheduleEarliestExpiry(now);
```

Comment:
I don't think this is the right thing to do here, but we shouldn't need to do this if the `ExpireNotification` action does `Promise.all` on the two dispatches, rather than awaiting one then awaiting the other